src/reconcile/rec_visibility.c

```diff
@@ -429,13 +436,24 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
      */
     if (!F_ISSET(r, WT_REC_LOOKASIDE | WT_REC_UPDATE_RESTORE))
         WT_ERR(__wt_set_return(session, EBUSY));
-    if (list_uncommitted && !F_ISSET(r, WT_REC_UPDATE_RESTORE))
+
+    /*
+     * If we have a list with an uncommitted update on it we still want to write out to the
+     * lookaside file in checkpoint.
+     */
+    if (list_uncommitted && !F_ISSET(r, WT_REC_UPDATE_RESTORE) && !F_ISSET(r, WT_REC_CHECKPOINT))
         WT_ERR(__wt_set_return(session, EBUSY));
 
     WT_ASSERT(session, r->max_txn != WT_TXN_NONE);
 
     WT_ERR(__rec_update_save(session, r, ins, ripcip, upd_select->upd, upd_memsize));
-    upd_select->upd_saved = true;
+
+    /*
```

Comment:
Can you elaborate this case in the comment?

Reply:
Unfortunately not without an extensive amount of time spent debugging test_las06, theres a case where eviction seems to have an insert list and still need to write content to lookaside. I attempted to elaborate a bit in the comment.